### ISSUE TITLE
PyTorch performance?

### ISSUE THREAD

**Issue by Jongchan** [hangzhaomit/semantic-segmentation-pytorch#9]
I'm about to run the code by myself, but just eager to know ASAP.

How is the performance after running your code? (Pixel acc, mIoU, etc)

**Comment by rosenfeldamir** [comment]
I ran the default settings myself, the accuracy is reported during training and is about 73%. The mean IOU was much lower, however, around 22, far lower than reported by the paper. The was done using the evaluation code provided.
Perhaps the default settings are not the ones that were used to generate the results in the paper.

**Comment by hangzhaomit** [comment]
@Jongchan @rosenfeldamir 
Thanks for the inquiry, I am still in the process of benchmarking all combinations of the models. Please keep an eye on the repo updates.
Here are the numbers I have now:
resnet34_dilated8 + c1_bilinear: Mean IoU 0.3104, Accuracy: 76.24%
resnet34_dilated8 + psp_bilinear: Mean IoU 0.3558, Accuracy: 77.72%

**Comment by rosenfeldamir** [comment]
Thansks, @hangzhaomit . Could I ask, is this using the default parameters in the code?
How many parameters did you use, and was the computation distributed?
I have read that PyTorch does not currently correctly support batch-normalization layers when training on multiple GPUs.

**Comment by hangzhaomit** [comment]
@rosenfeldamir Yes, I just added this as an important note in README. In order to deal with this issue, I pretrain the model to get good parameters for BN.
